packages/contracts/test/contracts/ovm/execution-manager/ExecutionManager.purity-checking.spec.ts

```diff
@@ -103,6 +103,8 @@ describe('Execution Manager -- Safety Checking', () => {
         AddThree,
         []
       )
+      console.log('Execution Manager Address:', executionManager.address)
+      console.log('AddThree Bytecode:', AddThree.bytecode)
```

Comment:
Oh should this be `log.debug(...)` instead?

Reply:
/ maybe this was left in by mistake?

Reply:
Yep, was a mistake. Fixed!